ooyala_player/public/js/ooyala_player.js

```diff
@@ -183,6 +183,17 @@ function OoyalaPlayerBlock(runtime, element) {
                     time: this.player.getPlayheadTime(),
                     player_state: this.player.getState()
                 });
+                console.log('fullscreenchanged')
+                var playing = this.player.isPlaying();
+                console.log('playing: ' + playing)
+                if (playing == true) {
```

Comment:
how is this functionality related to fullscreen event?

Reply:
Dsicrepency only occurs at fullscreen change
Check at https://stage.mckinsey.edx.org/courses/UI/Refresh/2019/lessons/i4x://UI/Refresh/chapter/3365eed34374428492519f2b669aa33f/module/i4x://UI/Refresh/vertical/2908dbb7725146ffa4139b3e266ff9e9
`rijjaasif`